ci/scripts/mask-rom-tests.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e -x
+
+# This doesn't install dependencies, but should otherwise behave the
+# same as what CI would do on a pull request
+DEFAULT="master"
+
+# use tests from a condonedlist
```

Comment:
What's the criteria for a test being on this list? Can we file an issue to get rid of the list altogether once we delete Meson?